Mathlib/Analysis/SpecialFunctions/Log/Deriv.lean

```diff
@@ -319,4 +319,18 @@ theorem hasSum_log_one_add_inv {a : ℝ} (h : 0 < a) :
     linarith
   · field_simp
 
+/-- Expansion of `log (a + 1)` as a series in powers of `a / (a + 2)`. -/
+theorem hasSum_log_one_add {a : ℝ} (h : 0 < a) :
+    HasSum (fun k : ℕ => (2 : ℝ) * (1 / (2 * k + 1)) * (a / (a + 2)) ^ (2 * k + 1))
+      (log (1 + a)) := by
+  have key := Real.hasSum_log_one_add_inv (inv_pos.mpr h)
```

Comment:
I think this result is still true when `a = 0` (in this case the infinite sum has only one term). Not sure what if `-1 < a`.

Also, maybe the `Real` in  `Real.hasSum_log_one_add_inv` can be omitted? (Same as the result below.)

Reply:
You can use `by_cases ha0 : a = 0` to give an alternative (possibly very simple) proof in `a = 0` case. I'm not sure the case `-1 < a < 0`, clearly it requires another proof...